### PR TITLE
fix rename bug: access(read/write/close) fd failed after rename an opend file

### PR DESCRIPTION
**Pull request by xuensheng**

### CODE REVIEW

**Comment by xuensheng** [comment]
result in fd leak

File: src/fdcache.cpp [hunk 1]
```diff
@@ -1893,6 +1893,17 @@ void FdManager::Rename(const std::string &from, const std::string &to)
     fent.erase(iter);
     ent->SetPath(to);
     fent[to] = ent;
+  } else {
+    for(fdent_map_t::iterator iter = fent.begin(); iter != fent.end(); ++iter){
+      if((*iter).second && (*iter).second->IsOpen()) {
+        // found opend fd in map
```
Comment (baiyubin) [inline src/fdcache.cpp]: - opend  -> opened

File: test/ut_test.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        os.rename(filename1, filename2)
+        
+        f = open(filename2, 'r')
+        f.seek(0, 0)
+        data2 = f.read()
+        f.close()
+
+        self.assertEqual(len(data1), len(data2))
+        self.assertEqual(data1, data2)
+
```
Comment (baiyubin) [inline test/ut_test.py]: - os.rename(filename, filename) 有问题吗？
- negative的case评估一下是否有需要。

File: test/ut_test.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+if __name__ == '__main__':
```
Comment (baiyubin) [inline test/ut_test.py]: ## ***CI 需要搞通过***

File: test/ut_test.py (hunk 2 above)
Comment (xuensheng) [reply test/ut_test.py]: os.rename(filename, filename)   这个是返回失败

**Pull request closed without merging**